third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -2892,6 +2892,19 @@ __RECORD_REPLAY_ARGUMENTS__.internal = {
   updateNextStackingContextId: (f) => { gNextStackingContextId = f(gNextStackingContextId); },
 };
 
+/** ###########################################################################
+ * Export JS API methods via `__RECORD_REPLAY__`.
+ * This is officially available for scripts in `eval*` commands to use.
+ * ##########################################################################*/
+Object.assign(__RECORD_REPLAY__, {
+  getOrCreateProtocolId(obj) {
```

Comment:
The fact that the ID is created on the fly is probably an implementation detail, so if we're doing this I'd be more for `getProtocolObjectId` or something. Maybe
* `getProtocolIdForObject`
* `getObjectFromProtocolId`
?

Reply:
I still have mixed feelings about taking this approach as an official API, but I recognize that it's probably fine as a first pass.

Reply:
Yeah, I had the same thought about the naming.  I'll do that.